src/analysis/PointsTo/PointsToFlowInsensitive.h

```diff
@@ -44,7 +44,9 @@ class PointsToFlowInsensitive : public PointerAnalysis
             n = (n->pointsTo.begin())->target;
         }
 
-        if (n->getType() == PSNodeType::FUNCTION)
+        if (n->getType() == PSNodeType::FUNCTION || 
+            n->getType() == PSNodeType::NULL_ADDR ||
+            n->getType() == PSNodeType::INVALIDATED) // TODO why?
```

Comment:
This change is irrelevant since the invalidate takes part only in flow-sensitive analysis.